include/mbgl/gfx/uniform_block.hpp

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include <string>
```

Comment:
This file doesn't include everything it needs to build the test runner on linux. Include unordered_map and memory here.